src/components/Header/HeaderNotification.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+export interface HeaderNotificationProps {
+  number?: number
+  url?: string
+}
+
+const HeaderNotificationComponent: React.FC<HeaderNotificationProps & InjectedProps> = ({
+  theme,
+  ...props
+}) => (
+  <Wrapper theme={theme}>
+    <Anchor
+      theme={theme}
+      number={props.number}
+      href={props.url ? props.url : '/notifications'}
+      aria-label="通知履歴"
+    >
+      {props.number && props.number > 10 ? '9+' : props.number}
```

Comment:
[ask] When props.number is 10, 10 is displayed, and when it is 11, '9+' is displayed. Is this correct?

Reply:
That’s incorrectly behavior. So I will fix that 💪